### PR TITLE
Fixes Analyzer Rules (Core Projects)

### PR DESCRIPTION
**Pull request by jtkech**

### CODE REVIEW

File: src/OrchardCore/OrchardCore.Abstractions/Caching/ISignal.cs [hunk 1]
```diff
@@ -16,7 +16,9 @@ public static class SignalExtensions
         /// <summary>
         /// Adds a Signal (if not already present) to be sent at the end of the shell scope.
         /// </summary>
+#pragma warning disable IDE0060 // Remove unused parameter
```
Comment (Piedone) [inline src/OrchardCore/OrchardCore.Abstractions/Caching/ISignal.cs:19]: Comment suppressions so it's apparent why there are there.

**Review by Piedone — COMMENTED** [review]
Great! Isn't this and the other PRs addressing https://github.com/OrchardCMS/OrchardCore/issues/7950 and https://github.com/OrchardCMS/OrchardCore/issues/12929.

**Comment by jtkech** [comment]
> Great! Isn't this and the other PRs addressing https://github.com/OrchardCMS/OrchardCore/issues/7950 and https://github.com/OrchardCMS/OrchardCore/issues/12929.

No, only based on the current `.editorconfig` and the default Analyzers we have, but yes we can think about more rules and more analyzers, e.g. about code blocks and comments formatting.

File: src/OrchardCore/OrchardCore.Abstractions/Caching/ISignal.cs (hunk 1 above)
Comment (jtkech) [reply src/OrchardCore/OrchardCore.Abstractions/Caching/ISignal.cs:19]: No problem, I can remove these warnings by using the discard symbol `_`

So I will remove all these pragmas ;)

**Comment by Piedone** [comment]
OK, thanks!

**Pull request merged**